mesonbuild/backend/backends.py

```diff
@@ -342,6 +342,13 @@ def object_filename_from_source(self, target, source, is_unity):
         assert isinstance(source, mesonlib.File)
         build_dir = self.environment.get_build_dir()
         rel_src = source.rel_to_builddir(self.build_to_src)
+
+        if (not self.environment.is_source(rel_src) or \
```

Comment:
__[Flake8]__

the backslash is redundant between brackets


<!-- Comment by [SideCI](https://sideci.com) -->